subtitles/vosk_transcriber.py

```diff
@@ -31,7 +36,12 @@ def generate(self, source: str, language: str) -> (str, str):
                                '-f', 'wav', "-"],
                               stdout=subprocess.PIPE).stdout as stream:
             if language in self.__recognizers:
-                return srt_to_vtt(self.__recognizers[language].SrtResult(stream)), language
+                mutex_vosk.acquire()
+                try:
+                    return self.__recognizers[language].SrtResult(stream), language
+                finally:
+                    mutex_vosk.release()
```

Comment:
I think that manually acquiring/releasing resources is likely error prawn. What do you think about this?
```suggestion
                with mutex_vosk:
                    return self.__recognizers[language].SrtResult(stream), language
```

Reply:
Much better. Didn't know that works too. Thanks!